unlock-app/src/config/app.ts

```diff
@@ -52,6 +54,9 @@ export const config = {
   ethPassApiKey: 'sk_live_h0pHRAZ2E6WTkNIrXvEzbEQN39Ftrp1p',
   walletConnectApiKey: '1535029cc7500ace23802e2e990c58d7', // https://cloud.walletconnect.com/app/project?uuid=7920be27-1e19-43a8-8f7d-cafbb00d4b80
   googleMapsApiKey: 'AIzaSyDp0Y4yQn6WtYEFEgRZg52EiDSgLwxzVMA',
+  googleClientId: process.env.NEXT_PUBLIC_GOOGLE_CLIENT_ID, // https://console.cloud.google.com/apis/dashboard
+  googleClientSecret: process.env.GOOGLE_CLIENT_SECRET, // https://console.cloud.google.com/apis/dashboard
+  coinbaseProjectId: process.env.NEXT_PUBLIC_COINBASE_PROJECT_ID, // https://portal.cdp.coinbase.com/
```

Comment:
since this is public you can set them directly here:
- staging: organizations/84d8b485-5433-4902-9765-3e5f11bf16d6/apiKeys/740902e7-eaee-4d60-a316-20690272e7e8
- prod: organizations/84d8b485-5433-4902-9765-3e5f11bf16d6/apiKeys/f2f8200c-77fc-4474-b5ec-fc5251f19bd8